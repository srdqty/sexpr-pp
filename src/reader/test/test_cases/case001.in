#;1 #;2 #;3 #;#;1 2
(#;a #;b #;(a b c))
1234
"hello world\n"
#\newline
#\space
#\tab
#\#
#\;
#\|
(#;#;1 2 a)
(a b)
(a . b)
(a b c d e f g h i j k l m n o p q r s t u v w x y z)
(a . (b . (c . (d . ()))))
(a . (b . (c . (d . e))))
(#;1 #;2 a #;1 #;2 . #;1 #;2
  (#;1 #;2 b #;1 #;2 . #;1 #;2 
    (#;1 #;2 c #;1 #;2 . #;1 #;2 
      (#;1 #;2 d #;1 #;2 . #;1 #;2 e #;1 #;2)
    #;1 #;2)
  #;1 #;2)
#;1 #;2)
#;1 #;2 #;3 #;#;1 2
