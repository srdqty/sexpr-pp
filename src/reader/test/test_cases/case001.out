()
1234
"hello world\n"
#\newline
#\space
#\tab
#\#
#\;
#\|
(a)
(a b)
(a . b)
(a b c d e f g h i j k l m n o p q r s t u v w x y z)
(a b c d)
(a b c d . e)
(a b c d . e)
<eof>
